lms/static/sass/discussion/utilities/_shame.scss

```diff
@@ -50,6 +54,11 @@
   @include padding-left(0);
 }
 
+.forum-nav-browse-menu-item {
```

Comment:
removed from `_shame`. Realized the old way had button and looks as text--black color. Now span is used, no need for color override.